### PR TITLE
Added Kafka - HTTP bridge examples

### LINKED ISSUE #17: Provide Kafka HTTP client examples
I would move my current Kafka HTTP client examples under this repo and adding the support for the tracing as well.

### PR DESCRIPTION
**Pull request by ppatierno**
This PR fixes #17

### CODE REVIEW

**Comment by ppatierno** [comment]
I am going to add Jaeger tracing with this PR as well, in order to fix #19

File: http/vertx/kafka-http-consumer/src/main/java/ConsumerApp.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import java.util.Map;
```
Comment (scholzj) [inline http/vertx/kafka-http-consumer/src/main/java/ConsumerApp.java:6]: Should we have the ASL headers in the Java files?

File: http/vertx/scripts/run.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (scholzj) [inline http/vertx/scripts/run.sh]: Is this script in any way different from the one we have for the regular Kafka clients? If not, should we have it just once?

**Review by scholzj — COMMENTED** [review]
I left some comments, but the code looks good.

**Review by scholzj — APPROVED** [review]

**Pull request merged**